docs/contributing.rst

```diff
@@ -8,46 +8,115 @@ For a more practical guide to the tedana development, please see our
+but somewhat distinct ideas related to multi-echo analysis. This scope includes
+collecting multi-echo data (Acquisition), combining those echoes together
+(Combining), with optional noise removal (Denoising), inspecting the outputs
+(Visualization) and answering multi-echo related questions (Community). In
+general, tedana accepts previously preprocessed data to produce outputs that
+are ready for further analyses. 
+
+Acquisition
+-----------
+
+While the development of multi-echo sequences is far beyond the current scope
+of tedana, the tedana community is committed to providing guidelines on current
+multi-echo implementations. This will include both specific instructions for
+how to collect multi-echo data for multiple vendors as well as details about
+what types of data have been collected thus far. These details are subject to
+change, and are intended to provide users with an idea of what is possible,
+rather than definitive recommendations.
+
+Our focus is on functional MRI, including both magnitude and phase data,
+however we understand that quantitative mapping has the potential to aid in
+data processing. Thus, we believe that some details on non-functional MRI
+acquisitions, such as detailed T2* mapping, may fall within the scope of
+tedana. Acquisition related details can be found in the tedana Documentation.
+
+Combining echoes
+----------------
+
+An early step in processing data collected with multiple echoes is the
+combination of the data into a single time series. We currently implement
+multiple options to combine multi-echo data and will add more as they continue
+to be developed. This is an area of active development and interest.
+
+Denoising
+---------
+
+tedana was developed out of a package known as [multi-echo ICA, ME-ICA, or MEICA](https://github.com/ME-ICA/me-ica )
+developed by Dr. Prantik Kundu. Though the usage of ICA for classification of
+signal vs noise components has continued in tedana, this is not a rule. The
+tedana community is open and encouraging of new denoising methods, whether or not they
+have a basis in ICA. 
+
+Specifically, we are interested in any method that seeks to use the information from multiple
+echoes to identify signal (defined here as BOLD signals arising from neural
+processing) and noise (defined here as changes unrelated to neural
+processing, i.e. motion, cardiac, respiration).
+
+tedana is primarily intended to work on volume data, that is, data that is
+still in structured voxel space. Surface-based denoising is not currently
+within the scope of tedana, but code should be written so that it is a
+possible option in the future.
+
+Currently tedana works on a single subject, run by run basis; however, methods
+that use information across multiple runs are welcome. 
+
+Visualization
+-------------
+
+Though tedana does not provide a GUI for inspecting results, it does produce
+figures as part of the processing stream. These figures are intended to help
```

Comment:
We should probably mention the interactive reports that were merged in not so long ago.